strings/italian.txt

```diff
@@ -0,0 +1,161 @@
+Non sono così sicuro di cosa faccia questa cosa, sembra un vecchio orologio da tasca. Il metallo è lavorato in modo fine, vale un sacco di soldi...
+Sembra uno strumento di rilevamento. Questi dispositivi del sistema di mappe costruiranno una mappa mentre cammini e ti indirizzeranno nella direzione del cancello iperspaziale più vicino.
+Ack! Stai attento con quelli! Quelle custodie sono altamente instabili!!!
+Un sofisticato trasmettitore radio, prova a usarlo vicino ad alcune macchine.
+Un'arancia! La leggenda narra che gli operai che hanno costruito questi punti di incontro siano stati qui per mesi alla volta... non volevano lo scorbuto!
+Un generatore portatile di porte iperspaziali... pensalo come... una sorta di polizza assicurativa se sei stordito.
+(USA E GETTA)
+bloccato, 
+bloccato, il Compare è in arrivo!
+ nemico rimasto
+ nemici rimasti
+Niente
+ho "
+"
+Inventorio pieno
+articoli
+punto di incontro libero
+punteggio 
+punteggio alto 
+punteggio alti
+tempo
+articoli collezionati % 
+
+articoli collezionati %
+punti di incontro 
+.
+Riprendi
+Connetti compare
+Impostazioni
+Script Console
+Salva e esci
+Ci vediamo più tardi...
+Scegli un obiettivo:
+Nessuna mappa stradale? Nessun problema! La rete di porte iperspaziali calcola un percorso verso la tua destinazione per il tuo viaggio attraverso una sequenza di punti di incontro. Decine di migliaia di punti di incontro sono stati costruiti in orbita attorno alla Terra e alla sua luna. Poiché i punti di incontro sono costantemente in movimento durante l'orbita, la rete di porte ricalcola continuamente il percorso verso la destinazione. Non visiterai mai due volte gli stessi punti di incontro! A causa della natura imprevedibile dei cancelli dell'iperspazio, il passaggio attraverso un cancello viene talvolta definito salto alla cieca.
+Vedi statistiche: 
+Lingua: 
+Contrasto: 
+Modalità notturna: 
+Inverti A/B: 
+predefinito
+Difficultà: 
+facile
+normale
+difficile
+sopravvivenza
+il livello deve essere pulito!
+Tasto Sparo: 
+separati
+combinati
+Registro Severità:
+Mostra Speedrun Clock: 
+Camera: 
+fissa
+tracciamento (basso)
+tracciamento (alto)
+Rimbombo Attivato: 
+si
+no
+menu disabilitato
+ m
+
+Compare collegato!
+Connessione compare fallita!
+Compare morto!
+Salute del compare cambiata in
+In attesa di compari...
+Sincronizzazione partita...
+ fps
+ tx
+ rx
+ tl
+ rl
+ sbr
+ lnsat
+richiesto: sistema Mappa
+Il viandante sconfitto
+Guardiano distrutto
+Hai sconfitto i gemelli
+Condotto infestato ripulito!
+Rete iperspaziale distrutta
+Compari in attesa alla porta...
+debug
+info
+avviso
+errore
+Sovraccarico rilevato!
+Acquista oggetti
+Vendere oggetti
+Acquista, prezzo:
+Prezzo di vendita:
+Informazioni sull'articolo
+Negozio dello spazzino
+acquistare
+vendere
+Aggiornamento richiesto!
+Sciame in avvicinamento!
+Il compare richiede un aggiornamento!
+globale
+Compare utilizza ibernazione!
+italiano
```

Comment:
this is the language that the game displays on the language selection screen, so we want "italian" translated to italian, instead of inglese

Reply:
let me know if this translation is correct, I don't know which gender conjugation is appropriate

Reply:
![BlindJump-67](https://user-images.githubusercontent.com/11904320/125371981-fbb81980-e34f-11eb-8945-8fe3c670c55b.png)

Reply:
it is perfect